pageserve/templates/index.html

```diff
@@ -33,9 +33,9 @@ <h2>Past posts</h2>
 <p>{{post.text}}</p>
 {% endif %}
 {% if post.files %}
-  {% for file in post.files %}
-  <img src="{{ file }}">
-  {% endfor %}
+{% for file in post.files %}
+<img src="{{ file }}">
+{% endfor %}
```

Comment:
Is there any way to get rid of the html flattening? If not it's not super important.

Reply:
Done https://github.com/knative-portability/large-events/pull/59/commits/68e9b6b82b01ca4e483e8941acd7ce8ba0b84255